packages/plugin-flex/src/__tests__/commands/flex/plugins/deploy.test.ts

```diff
@@ -68,6 +68,7 @@ describe('Commands/FlexPluginsDeploy', () => {
   const OLD_ENV = process.env;
 
   beforeEach(() => {
+    jest.setTimeout(10000);
```

Comment:
I don't think this needs to be beforeEach? Just add it once inside the `describe` method?